packages/sdk/src/solana/types/programs/index.ts

```diff
@@ -27,7 +27,7 @@ export const CommonContractOutputSchema = CommonContractSchema.extend({
  */
 export const CreatorInputSchema = z.object({
   address: z.string(),
-  sharePercentage: z.number(),
+  share: z.number(),
```

Comment:
only annoying thing is that now its not clear if the unit is bps or percentage. Maybe we can add a transform here to ensure the value is between 0 and 100 ? you can copy what we did for BasisPointsSchema

Reply:
used the PercentSchema we already had